### PR TITLE
160 save the widgets local filters in the layout

### PR DESCRIPTION
**Pull request by FadyCoding**
## Describe your changes
Saved the widgets local filters whenever loading and saving a new layout is done.
## Issue ticket number and link
Issue ticker 160
https://github.com/debiai/DebiAI/issues/160
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have increased the version number in the `frontend/package.json` and `backend/swagger.yaml` files
- [x] I have checked that Black, Flake8, Prettier and Cspell are passing

### CODE REVIEW

File: frontend/src/components/debiai/dataAnalysis/DataAnalysis.vue [hunk 1]
```diff
@@ -569,9 +578,10 @@ export default {
           height: component.height,
           widgetKey: component.widgetKey,
           config: component.config,
+          localFilters: component.localFilters,
         });
       });
-
+      // Save the body into a json file
```
Comment (Tomansion) [inline frontend/src/components/debiai/dataAnalysis/DataAnalysis.vue:584]: To remove

**Review by Tomansion — CHANGES_REQUESTED** [review]
Well done
Somme last changes and you're good

File: backend/swagger.yaml [hunk 2]
```diff
@@ -706,8 +706,8 @@ paths:
           schema:
             type: object
 
-  /data-providers/{dataProviderId}/projects/{projectId}/selections/{selectionId}:
-    get:
+  ? /data-providers/{dataProviderId}/projects/{projectId}/selections/{selectionId}
+  : get:
```
[the remark this answers is not in the document]
Comment (FadyCoding) [reply backend/swagger.yaml:710]: It added itself on its own. Does not work with it, idk why.

File: backend/swagger.yaml [hunk 3]
```diff
@@ -524,8 +524,8 @@ paths:
         404:
           description: model or project doesn't exist
 
-  /data-providers/{dataProviderId}/projects/{projectId}/models/{modelId}/getModelResults:
-    post:
+  ? /data-providers/{dataProviderId}/projects/{projectId}/models/{modelId}/getModelResults
+  : post:
```
[the remark this answers is not in the document]
Comment (FadyCoding) [reply backend/swagger.yaml:528]: It added itself on its own. Does not work with it, idk why.

**Review by Tomansion — APPROVED** [review]

**Pull request merged**